app/kumactl/cmd/install/install_control_plane_test.go

```diff
@@ -93,7 +93,7 @@ var _ = Describe("kumactl install control-plane", func() {
 			// then
 			Expect(actual).To(MatchYAML(expected))
 			// and
-			actualManifests := data.SplitYAML(data.File{Data: expected})
+			actualManifests := data.SplitYAML(data.File{Data: actual})
```

Comment:
Good catch! I was having problems with this last time I edited the install script and was wondering what is going on.